lib/galaxy/config/sample/job_conf.xml.sample_advanced

```diff
@@ -1061,12 +1061,12 @@
             <param id="docker_enabled">true</param>
             <param id="docker_memory">512</param>
             <param id="docker_cpu">2</param>
-            <param id="volumes">/directory/</param>
+            <param id="volumes">host_path:container_path:mode</param>
             <!-- Directory which is mounted to the container and is parent of
                  the  `job_working_directory`, `file_path`, `new_file_path`
-                 directories. Directories of the data used by tools are
-                 included as well.-->
-            <param id="max_retries">2</param>
+                 directories. Other directories of the data used by tools can
+                 be mounted as well, separated by commas.-->
+            <param id="max_retries">0</param>
```

Comment:
Any chance I can get you to update the corresponding docs in the sample YAML file (https://github.com/galaxyproject/galaxy/blob/dev/test/unit/jobs/job_conf.sample_advanced.yml). It copied a lot of the comments out of this sample XML but I haven't gotten around to replacing the XML version with the YAML version - but the YAML will be the go to at some point soon.

Reply:
This goes without saying! Next time I will update it by default.